astroquery/mast/tests/test_mast_remote.py

```diff
@@ -541,6 +541,40 @@ def test_get_cloud_uris(self):
 
         assert len(uris) > 0, f'Products for OBSID {test_obs_id} were not found in the cloud.'
 
+        # check for warning if no data products match filters
+        with pytest.warns(NoResultsWarning):
+            Observations.get_cloud_uris(products,
+                                        extension='png')
+
+    def test_get_cloud_uris_query(self):
+        pytest.importorskip("boto3")
+
+        # enable access to public AWS S3 bucket
+        Observations.enable_cloud_dataset()
+
+        # get uris with other functions
+        obs = Observations.query_criteria(target_name=234295610,
+                                          provenance_name="SPOC",
```

Comment:
This triggers a test failure. Could you please run and fix the remote tests?

``pytest -P mast -R``?

Reply:
That's strange, the tests pass on my local machine. What was the error? There's been some slowdowns on the MAST servers recently, so that might be the cause.